### PR TITLE
Launch Newsroom Project.

### PR DESCRIPTION
**Pull request by tatwell**
Trello Card: https://trello.com/c/nrglr3k9

### CODE REVIEW

**Comment by tatwell** [comment]
@kaylaziegler Which patch version of Rails are you using? I'm on `5.1.4`:

```
$ rails -v
Rails 5.1.4
```

That *could* make a difference although I doubt it. But I have no other ideas at moment why it would work for me and not for you.

**Comment by kaylaziegler** [comment]
@tatwell I'm using the same, 5.1.4.

**Comment by tatwell** [comment]
@kaylaziegler When I run `bundle exec rake test`, I get the same error. Interesting.

**Comment by tatwell** [comment]
@kaylaziegler I think I've fixed it. The issue, I believe, was that I didn't require `test_helper` with the controller tests which resulted in app not being loaded properly (and missing route helpers.)

Please pull and try again.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+    # Copy dist versions of config files into place
+    cp -v config/secrets.yml{.dist,}
+    cp -v config/database.yml{.dist,}
+
+
+## Development
+### Running local server
+To start the local server, use webrick (puma is not configured to run locally):
+
+    rails s webrick
```
Comment (kaylaziegler) [inline README.md]: Just an FYI, I am able to run puma locally using just `rails s`. Doesn't this only apply when we set it up using ansible?

Comment (tatwell) [reply README.md]: You're right. I'll update this in master.

**Pull request merged**